charts/stable/qbittorrent/questions.yaml

```diff
@@ -24,7 +24,8 @@ questions:
           description: Enable and configure Gluetun Port Forwarding Automation.
           schema:
             type: boolean
-            default: false
+            # Change back to false before merge
+            default: true
```

Comment:
add a `ci/cron-values.yaml` file with this enabled, so it always runs on CI with this enabled ;)

Reply:
Did not seem to run anyway. Just checked logs.

Reply:
well...questions typically dont turn stuff on in testing here; thats values where you have it disabled.

Reply:
Think I did it right